app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -8,6 +8,9 @@ class Vmware < ComputeResource
     validates :user, :password, :server, :datacenter, :presence => true
     before_create :update_public_key
 
+    alias_attribute :server, :url
+    alias_attribute :datacenter, :uuid
```

Comment:
does this works for setters? I would assume you would need to alias 'datacenter=' to 'uuid=' ?